src/imp/security_framework.rs

```diff
@@ -7,6 +7,7 @@ use self::security_framework::base;
 use self::security_framework::certificate::SecCertificate;
 use self::security_framework::identity::SecIdentity;
 use self::security_framework::import_export::{ImportedIdentity, Pkcs12ImportOptions};
+use self::security_framework::os::macos::import_export::Pkcs12ImportOptionsExt;
 use self::security_framework::secure_transport::{
     self, ClientBuilder, SslConnectionType, SslContext, SslProtocol, SslProtocolSide,
 };
```

Comment:
Doesn't set_peer_id need to be called here?

Reply:
It's called internally in https://github.com/kornelski/rust-security-framework/pull/95/files#diff-0c189053dd842cb8ae78ca32925584a8R1359, since we're using the `ClientBuilder` rather than manually building an `SslContext` (which is the one that has a `set_peer_id` method).